test/commands/CodeblockCommandTest.ts

```diff
@@ -20,6 +20,13 @@ describe("CodeblockCommand", () => {
 
 			expect(command.getDescription()).to.equal("Shows a tutorial on how to use Discord's codeblocks.");
 		});
+
+		it("creates a command with correct aliases", () => {
+			const command = new CodeblockCommand();
+
+			expect(command.getAliases().includes("codeblocks")).to.equal(true);
+			expect(command.getAliases().includes("cb")).to.equal(true);
+		});
```

Comment:
We should really mock a command for testing rather than using an instance of `CodeblockCommand`. What happens if the logic changes for this command or it gets removed?